src/components/RichTextEditor/serializer/utils/reactCssPropsToCss.ts

```diff
@@ -1,8 +1,9 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { AnyObject } from '@udecode/plate';
 import { convertCamelCaseToKebabCase } from './convertCamelCaseToKebabCase';
 
-export const reactCssPropsToCss = (props?: any): string => {
+export const reactCssPropsToCss = (props?: AnyObject): string => {
```

Comment:
🙈 

Isn't it `DesignTokens`?

Reply:
It's of type `CSSProperties`. I've updated it now. Thanks for the hint 👍